data/Language/Flow.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: Flow
+full_name: Flow
+wikidata: Q59413589
+# versions: v0.1.0 - v0.87.0
+creation_date: 2014
+extensions:
+  - .js.flow
+  - /flow-typed/*.js
```

Comment:
that doesnt look like an extension.  you need to do some modelling

Reply:
Yeah, this meant that the `.js` files under `flow-typed` folder would be equivalent to `.js.flow`.
Is it better to remove that, altogether?